doc/development_setup.md

```diff
@@ -0,0 +1,34 @@
+# Development Setup
+
+This document should get you up to speed and give you a working development environment for
+`transcribee`. The instructions in this document may not be used for production installations!
+
+The transcribee project uses a development-setup managed by `nix`. This allows us to all use the
+same software & versions conviniently. Thus, [you need to have a working nix installation on your
```

Comment:
This is a pretty big requirement... I think it would be feasible to install 7 dependencies by hand.